src/main/java/net/querz/mcaselector/io/mca/ChunkData.java

```diff
@@ -13,6 +14,7 @@ public boolean relocate(Point3i offset) {
 	}
 
 	private boolean relocateChunk(Chunk c, Point3i offset) {
+		// MAINTAINER why do we check if DataVersion exists, shouldn't that be a (fast-failing) irrecoverable corruption?
```

Comment:
In other words: why is it okay to silently ignore this chunk and continue the relocation operation if `DataVersion` is missing?

Reply:
That is true. It could log a warning in case of failure to find the `DataVersion` tag, but only if the chunk has any data (`c.getData() != null`), otherwise it should just ignore that chunk.